src/reaper_osara.cpp

```diff
@@ -153,6 +153,12 @@ string formatTime(double time, TimeFormat timeFormat, bool isLength,
 			timeFormat = TF_MEASURE;
 		}
 	}
+	if (includeProjectStartOffset) {
+		if (timeFormat != TF_MEASURE && timeFormat != TF_SAMPLE) {
+			ReaProject* proj = EnumProjects(-1, nullptr, 0);
+			time += GetProjectTimeOffset(proj, false);
```

Comment:
For most REAPER functions, you can just pass nullptr to choose the current project. Does that work here?